Library/Homebrew/version.rb

```diff
@@ -408,6 +408,11 @@ def self._parse(spec)
     # e.g. http://www.ijg.org/files/jpegsrc.v8d.tar.gz
     m = /\.v(\d+[a-z]?)/.match(stem)
     return m.captures.first unless m.nil?
+
+    # date-based versioning
+    # e.g. ltopers-v2017-04-14.tar.gz
```

Comment:
(in https://github.com/Homebrew/brew/blob/master/Library/Homebrew/test/version_spec.rb)